eslint.config.js

```diff
@@ -104,7 +104,10 @@ export default tsEslint.config(
             'prefer-template': 'error',
             radix: 'error',
             'require-await': 'error',
-            'sort-imports': ['error', { allowSeparatedGroups: true }],
+            'sort-imports': [
```

Comment:
What's with the relaxation of the rules? These were purposely (overly) strict to enforce convention so that there is no need to discuss formatting in PRs.